site/pages/disruption-detail/[disruptionId].page.tsx

```diff
@@ -600,16 +600,20 @@ const DisruptionDetail = ({
                                 </div>
                             ))}
                         </div>
-                        <Link
-                            role="button"
-                            href={{
-                                pathname: `${TYPE_OF_CONSEQUENCE_PAGE_PATH}/${disruption.disruptionId}/${nextIndex}`,
-                                query: { return: DISRUPTION_DETAIL_PAGE_PATH },
-                            }}
+                        <button
```

Comment:
Actually ignore what I said sorry, a button will cause issues with submitting the form, this will work similar to what you had before:

```tsx
<Link
    href={{
        pathname: `${TYPE_OF_CONSEQUENCE_PAGE_PATH}/${disruption.disruptionId}/${nextIndex}`,
        query: { return: DISRUPTION_DETAIL_PAGE_PATH },
    }}
    className={`govuk-button mt-2 govuk-button--secondary ${
        disruption.consequences && disruption.consequences.length >= 10
            ? "pointer-events-none govuk-button--disabled"
            : ""
    }`}
    >
    Add another consequence
</Link>
```

Reply:
done